ingestion/src/kldata.rs

```diff
@@ -38,6 +38,29 @@ pub enum ParseError {
     UnrecognisedParamCode(String),
 }
 
+/// FIXME: these params are scalar in Stinfosys, but are not when coming from Obsinn.
+pub const SPECIAL_CASES: [&str; 10] = [
+    // METAR params that come in as 'xxL' and 'xxR', where 'x' is a numeric character.
+    // We need to decide how to treat them (Kvalobs silently discards them apparently)
+    // Or if they need to be changed in Stinfosys
+    "X1R",
+    "X2R",
+    "X3R",
+    "WS",
+    "WS2",
+    // seems to be a number most of the times, but gets an 'a' every once in a while.
+    // Maybe it's in hex format?
+    "W1",
+    // with value "N70.18664E30.05277", can maybe be separated into lat and lon params?
+    "STN#META##LatitudeLongitude",
+    // with value "Flatetemperatur"
+    "TSS",
+    // with value "windspeed10m"
+    "FF_01",
+    // with value "winddir10m"
+    "DD_01",
```

Comment:
These are not floats?

Reply:
I posted the values they were sent with above, I at least have no idea how to parse those as floats.